.travis.yml

```diff
@@ -6,7 +6,6 @@ install:
 - bundle install --retry=3
 script:
 - bundle exec rspec
-- bundle exec rubocop --config .rubocop.yml
```

Comment:
I will have to remember to put this line back after @sylyu merged the rubocop config